### PR TITLE
Allow Customizing OpenMapOptions in LocationPickerField

### PR DESCRIPTION
**Pull request by xmine64**

### CODE REVIEW

File: lib/src/location_picker_field.dart [hunk 1]
```diff
@@ -116,24 +119,29 @@ class __BaseFormFieldState<T> extends State<_BaseFormField<T>> {
...
-                  textAlign: widget.textAlign,
-                  textAlignVertical: widget.textAlignVertical,
-                  isFocused: _isFocused,
-                  isHovering: _isHovering,
-                  expands: widget.expands,
-                  child: widget.display(field.value),
+          clipper: ShapeBorderClipper(
+              shape:
+                  effectiveDecoration.border ?? const RoundedRectangleBorder()),
+          child: Padding(
```
Comment (mu-dawood) [inline lib/src/location_picker_field.dart]: why you are adding padding here

File: lib/src/location_picker_field.dart [hunk 2]
```diff
@@ -309,7 +322,8 @@ class MultiOpenMapPicker extends StatelessWidget {
           return OpenMapOptions();
         } else {
           return OpenMapOptions.bounds(
-            bounds: LatLngBounds.fromPoints(list.map((e) => e.toLatLng()).toList()),
```
Comment (mu-dawood) [inline lib/src/location_picker_field.dart]: add it also in `MultiOpenMapPicker`

File: lib/src/location_picker_field.dart (hunk 1 above)
Comment (xmine64) [reply lib/src/location_picker_field.dart]: Use the field with OutlinedInputBorder then you'll see.

Comment (mu-dawood) [reply lib/src/location_picker_field.dart]: Screen Shot 2022-04-20 at 8 39 50 PM


as you can see without the padding it has the same behaviour  of text field

Comment (mu-dawood) [reply lib/src/location_picker_field.dart]: its not the package responsibility to add spacing between fields another may need a larger space than you or don't need space at all

Comment (xmine64) [reply lib/src/location_picker_field.dart]: Add some space between fields in this form then select a location in a field and see the result.

Comment (xmine64) [reply lib/src/location_picker_field.dart]: I should've opened an issue for it with screenshots

Comment (mu-dawood) [reply lib/src/location_picker_field.dart]: you can open the issue to see what is wrong and found a solution for it

**Pull request merged**